solutions/platinum/cses-1706.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: cses-1706
+source: CSES
+title: School Excursion
+author: Brebenel Mihnea
+---
+
+## Explanation
+
+The solution is based on DSU ([Disjoin Set Union](https://usaco.guide/gold/dsu?lang=cpp))
+For example: If 2 and 3 want  the same thing and 3 and 4 want the same think, so it means that 2, 3 and 4 want the same thing. After we determine the components, we need to find out every combination of them. To do that, we use a knapsack DP (you can find a good tutorial [here](https://usaco.guide/gold/knapsack?lang=cpp)). But it would take $O(N ^ 2)$, which will cause TLE. In order to speed it up, we use a bitset as a knpasack. The final complexity is $\mathcal{O}(N\log^2N)$
```

Comment:
do the same thing for the link here, and end sentences with periods. use `\mathcal O` for all time complexities

Reply:
Done

Reply:
knapsack link wasn't updated

Reply:
Sorry ! I updated it

Reply:
there are still sentences that don't end with periods. also, think -> thing

Reply:
It's ok now ?